src/+otp/+constrainedpendulum/ConstrainedPendulumParameters.m

```diff
@@ -0,0 +1,11 @@
+classdef ConstrainedPendulumParameters
+    %CONSTRAINEDPENDULUMPARAMETERS
+    properties
+        %Gravity defines the magnitude of acceleration towards the ground
+        Gravity %MATLAB ONLY: (1,1) {mustBeNonnegative}
+        %Mass defines the mass of the pendulum
+        Mass %MATLAB ONLY: (1,1) {mustBeNonnegative}
```

Comment:
Sharper validation: `{mustBeReal, mustBeFinite, mustBePositive}`

Reply:
Better yet, it should match https://github.com/ComputationalScienceLaboratory/ODE-Test-Problems/blob/3d4cb2262931eaf261ca3061479b81928b1ef419/src/+otp/+pendulum/PendulumParameters.m

Reply:
Good idea